tests/Workflow/Integration/Test_UserJobs.py

```diff
@@ -18,127 +18,128 @@
 from DIRAC.Interfaces.API.Job import Job
 from DIRAC.Interfaces.API.Dirac import Dirac
 
-class UserJobTestCase( IntegrationTest ):
+
+class UserJobTestCase(IntegrationTest):
   """ Base class for the UserJob test cases
   """
-  def setUp( self ):
-    super( UserJobTestCase, self ).setUp()
+
+  def setUp(self):
+    super(UserJobTestCase, self).setUp()
 
     self.d = Dirac()
-    self.exeScriptLocation = find_all( 'exe-script.py', '..', '/DIRAC/tests/Workflow' )[0]
-    self.helloWorld = find_all( "helloWorld.py", '..', '/DIRAC/tests/Workflow' )[0]
-    self.mpExe = find_all( 'testMpJob.sh', '..', '/DIRAC/tests/Utilities' )[0]
+    self.exeScriptLocation = find_all('exe-script.py', '..', '/DIRAC/tests/Workflow')[0]
```

Comment:
from DIRAC import rootPath
Could you please use rootPath instead?
Because it will not work if we are in wrong directory